crates/pipeline_manager/src/db/mod.rs

```diff
@@ -1426,9 +1431,10 @@ impl ProjectDB {
         };
         let mgr = Manager::from_config(config, NoTls, mgr_config);
         let pool = Pool::builder(mgr).max_size(16).build().unwrap();
-        let client = pool.get().await?;
-
-        client.simple_query(include_str!("ddl.sql")).await?;
+        let mut client = pool.get().await?;
+        let pg_client = client.deref_mut().deref_mut();
+        let report = embedded::migrations::runner().run_async(pg_client).await?;
```

Comment:
You should be able to do `run_async(&mut **pg_client)` instead of all the `deref_mut` stuff.